.github/workflows/astra.yml

```
@@ -59,6 +59,6 @@ jobs:
 
     - name: Run tests
       env:
-        ASTRA_API_ENDPOINT: ${{ secrets.ASTRA_API_ENDPOINT }}
-        ASTRA_TOKEN: ${{ secrets.ASTRA_TOKEN }}
+        ASTRA_DB_API_ENDPOINT: ${{ secrets.ASTRA_DB_API_ENDPOINT }}
+        ASTRA_DB_APPLICATION_TOKEN: ${{ secrets.ASTRA_DB_APPLICATION_TOKEN }}
```

Let's keep the old secrets name for now, it's not user-facing and it'll spare setting new secrets for now

```suggestion
        ASTRA_DB_API_ENDPOINT: ${{ secrets.ASTRA_API_ENDPOINT }}
        ASTRA_DB_APPLICATION_TOKEN: ${{ secrets.ASTRA_TOKEN }}
```

---

Great point, that was a silly oversight on my part! Yeah, since these are just repo secrets absolutely no reason they need to match. updating.